app/src/main/res/values/colors.xml

```diff
@@ -32,4 +32,5 @@
 
     <color name="session_header">@color/color_primary</color>
 
+    <color name="speaker_social_button_color">@color/color_primary</color>
```

Comment:
This color name is out of nowhere. Pretty sure it is not used anywhere in speaker screen.